### PR TITLE
adding post code to region function

### PR DESCRIPTION
**Pull request by colins44**
we need to have the postcode function within both auth and sailthru

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 var currencyData = require('./currency-data').currencies
 var regionData = require('./region-data')
 var flights = require('./lib/flights')
+var regionMapper = require('./lib/regionMapper')
```
Comment (joshuacullenlux) [inline index.js]: It's a bit confusing that they're both called region. Maybe call this cityPostCodeMapper?

File: lib/regionMapper.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+var PostCodeMap = [
```
Comment (joshuacullenlux) [inline lib/regionMapper.js]: Is this going to be used in the frontend by the customer portal?

Might be a good idea to create your own node module called lib-postcodes?

Comment (colins44) [reply lib/regionMapper.js]: Currently we only have one function within this and the function does map postcodes to regions, so i think its valid to have within here

**Comment by colins44** [comment]
@eddiemcleanlux I suggest marketingRegionFromPostCode()

Turns out that the email marketing team also just call them "regions"

**Pull request merged**